packages/channels/botpress-channel-messenger/config.json

```diff
@@ -14,30 +14,30 @@
     Optional settings
   *************/
 
-  "graphVersion": "2.12",
-  "enabled": true,
+  "graphVersion": "2.12", // ok
```

Comment:
Are these left overs?

Reply:
@slvnperron yes, it should be removed. But I've left it to mark places where I have troubles.

Reply:
We should remove / address these before merging